app/api/v2/handlers/operation_api.py

```diff
@@ -34,39 +34,53 @@ def add_routes(self, app: web.Application):
+                                      '`BaseGetAllQuerySchema` in the request body to filter.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(OperationSchema(many=True, partial=True))
     async def get_operations(self, request: web.Request):
         operations = await self.get_all_objects(request)
         return web.json_response(operations)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Retrieve an operation by operation id',
+                          description='Retrieve one CALDERA operation from memory based on the operation id (String '
+                                      'UUID).  Use fields from the `BaseGetOneQuerySchema` in the request body to add '
+                                      '`include` and `exclude` filters.')
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(OperationSchema(partial=True))
     async def get_operation_by_id(self, request: web.Request):
         operation = await self.get_object(request)
         return web.json_response(operation)
 
     @aiohttp_apispec.docs(tags=['operations'],
-                          summary='Required nested schema fields are as follows: "adversary.adversary_id", '
-                                  '"planner.planner_id", and "source.id".')
-    @aiohttp_apispec.request_schema(OperationSchema())
+                          summary='Create a new CALDERA operation record',
+                          description='Create a new CALDERA operation using the format provided in the '
+                                      '`OperationSchema`. Required nested schema fields are as follows: '
+                                      '"adversary.adversary_id", "planner.planner_id", and "source.id"')
+    @aiohttp_apispec.request_schema(OperationSchema)
     @aiohttp_apispec.response_schema(OperationSchema)
     async def create_operation(self, request: web.Request):
         operation = await self.create_object(request)
         return web.json_response(operation.display)
 
-    @aiohttp_apispec.docs(tags=['operations'])
-    @aiohttp_apispec.request_schema(OperationSchema(partial=True, only=['state',
-                                                                        'autonomous',
-                                                                        'obfuscator']))
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Update fields within an operation',
+                          description='Update one CALDERA operation in memory based on the operation id (String '
+                                      'UUID). Any fields in the operation object may be edited in the request body '
+                                      'using the `OperationSchema`.')
+    @aiohttp_apispec.request_schema(OperationSchema(partial=True))
```

Comment:
This is the only endpoint where this specific change needs to be made, but can you add back the `only=['state', 'autonomous', 'obfuscator']` to the `request_schema`? The `store()` method of `c_operation.py` is called whenever we issue a PATCH/update request, and the only fields that can be updated in that method are state, autonomous, and obfuscator